configsuite/types.py

```diff
@@ -167,7 +167,7 @@ def _is_bool(x):
     return isinstance(x, bool)
 
 
-@validator_msg("Is x a date")
+@validator_msg("Is x a valid ISO8601(Y-m-d) date format")
```

Comment:
Should we even make it `YYYY-mm-dd` to make it even clearer what format is expected?

Reply:
Good suggestion!